varats/varats/data/reports/blame_report.py

```diff
@@ -37,20 +42,28 @@ def create_blame_inst_interactions(
     ) -> 'BlameInstInteractions':
         """Creates a `BlameInstInteractions` entry from the corresponding yaml
         document section."""
-        base_hash = str(raw_inst_entry['base-hash'])
-        interacting_hashes: tp.List[str] = []
+        base_commit, *base_repo = str(raw_inst_entry['base-hash']).split('-')
```

Comment:
```suggestion
        base_commit, *base_repo = str(raw_inst_entry['base-hash']).split('-', maxsplit=1)
```
This could avoid problems with repo-names containing a "-".